### PR TITLE
added tictactoe game

### PR DESCRIPTION
**Pull request by SheryarAmir**

### CODE REVIEW

**Comment by SheryarAmir** [comment]
ticket : https://linear.app/northnine/issue/DEV-35/tic-tac-toe

File: sheryar/tictactoe/javascript.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+const checkWinner = () => {
+  for (let pattern of winPatterns) {
+    let pos1val = boxes[pattern[0]].innerText;
+    let pos2val = boxes[pattern[1]].innerText;
+    let pos3val = boxes[pattern[2]].innerText;
+
+    if (pos1val !== "" && pos2val !== "" && pos3val !== "") {
+      if (pos1val === pos2val && pos2val === pos3val) {
+        showWinner(pos1val);
+        playMusic("/sound/winning sound.mp3");
```
Comment (naveedalirehmani) [inline sheryar/tictactoe/javascript.js:64]: repalce `playMusic("/sound/winning sound.mp3");` with this `playMusic("./sound/winning sound.mp3");`

File: sheryar/tictactoe/javascript.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+};
+
+const showWinner = (winner) => {
+  msg.innerText = `Congratulations! your the winner of the match ${winner} `;
+  msgContainer.classList.remove("hide");
+  disableBoxes();
+};
+
+const showDraw = () => {
+  msg.innerText = "The game is a draw trty again!";
```
Comment (naveedalirehmani) [inline sheryar/tictactoe/javascript.js:50]: fix typo.

File: sheryar/tictactoe/javascript.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
+const boxes = document.querySelectorAll(".box");
+const resetBtn = document.querySelector(".reset-btn");
+const newGameBtn = document.querySelector("#new-btn");
+const msgContainer = document.querySelector(".msg-container");
+const msg = document.querySelector("#msg");
```
Comment (naveedalirehmani) [inline sheryar/tictactoe/javascript.js:5]: maybe use a different variable names, something more explicit.

**Pull request merged**